daemon/openrazer_daemon/hardware/mouse.py

```diff
@@ -1928,6 +1928,33 @@ class RazerBasiliskV3XHyperSpeed(__RazerDevice):
     DPI_MAX = 18000
 
 
+class RazerDeathAdderV4ProWired(__RazerDevice):
+    """
+    Class for the Razer DeathAdder V4 Pro (Wired)
+    """
+    EVENT_FILE_REGEX = re.compile(r'.*usb-Razer_DeathAdder_V4_Pro_E408D86F2BC99521-if0(1|2)-event-kbd')
+
+    USB_VID = 0x1532
+    USB_PID = 0x00BE
+    METHODS = ['get_device_type_mouse', 'max_dpi', 'get_dpi_xy', 'set_dpi_xy', 'get_dpi_stages', 'set_dpi_stages',
+               'get_poll_rate', 'set_poll_rate',
```

Comment:
The maximum poll rate just needs `'get_supported_poll_rates'` adding here. The daemon isn't actually telling apps about `POLL_RATES` without it.